logging/src/main/java/org/jboss/as/logging/formatters/PatternFormatterResourceDefinition.java

```diff
@@ -111,6 +113,9 @@ public boolean isMarshallable(final AttributeDefinition attribute, final ModelNo
         @Override
         public void performRuntime(final OperationContext context, final ModelNode operation, final ModelNode model, final LogContextConfiguration logContextConfiguration) throws OperationFailedException {
             final String name = context.getCurrentAddressValue();
+            if (name.endsWith(DEFAULT_FORMATTER_SUFFIX)) {
+                throw new IllegalArgumentException("Formatter name cannot end with '" + DEFAULT_FORMATTER_SUFFIX + "'");
```

Comment:
This needs to be internationalized and I think an `OperationFailedException` would be a better one to throw.

Reply:
How do I do that? Should I turn this into a new message in LoggingLogger?

Reply:
Yes. You add a method to the `LoggingLogger` that returns an `OperationFailedException`.